.github/workflows/main.yml

```diff
@@ -26,16 +26,30 @@ jobs:
           - python-version: 3.9
             env:
               TOXENV: py
+          - python-version: "3.10"
+            env:
+              TOXENV: py
+          - python-version: "3.10"
+            env:
+              TOXENV: latest
+          - python-version: "3.11"
+            env:
+              TOXENV: py
+          - python-version: "3.11"
+            env:
+              TOXENV: latest
```

Comment:
I have not seen these `latest` tox environments before, I do not see `latest` defined at https://github.com/scrapinghub/number-parser/blob/master/tox.ini, and the CI output suggests they cause `py` to be used. So I suspect we can remove 2 jobs from here.